src/universe/math/geometry.py

```diff
@@ -7,15 +7,15 @@
 
 class Point2d:
     """
-    Represents a 2 dimensional point.
+    Represents a 2-dimensional point.
     """
 
     def __init__(self, x: float, y: float):
         self.x = x
         self.y = y
 
-    def __str__(self):
-        return f'{self.x=}, {self.y=}'
+    def __repr__(self):
+        return f'({self.x}, {self.y})'
```

Comment:
Instead of converting the __str__ method into the __repr_ method, lets keep both.
The purpose of the first one is to return an easy to read string, while the other supposed to return a string that, when passed to eval() would return the original object (same by value) or very similar.
```suggestion
            def __str__(self):
        return f"({self.x}, {self.y})"

    def __repr__(self):
        return f"Point2d({self.x}, {self.y})"
```